test/bin/test-citgm.js

```diff
@@ -2,13 +2,17 @@
 
 var test = require('tap').test;
 
-var spawn = require('../../lib/spawn');
+var spawn = require('child_process').spawn;
```

Comment:
Ok, I think you are right. I'll drop this change in `test-citgm.js` and leave it for another PR